deployable/java/src/main/java/envtest/deployable/Snippets.java

```diff
@@ -63,15 +61,19 @@ public void simplelog(Map<String,String> args){
         Severity severity = getSeverity(severityString);
 
         // Instantiates a client
-        Logging logging = LoggingOptions.getDefaultInstance().getService();
-        LogEntry entry =
-            LogEntry.newBuilder(StringPayload.of(logText))
-                .setSeverity(severity)
-                .setLogName(logName)
-                .setResource(MonitoredResource.newBuilder("global").build())
-                .build();
+        try (Logging logging = LoggingOptions.getDefaultInstance().getService()) {
+            LogEntry entry =
+                LogEntry.newBuilder(StringPayload.of(logText))
+                    .setSeverity(severity)
+                    .setLogName(logName)
+                    .setResource(MonitoredResource.newBuilder("global").build())
+                    .build();
 
-         //Writes the log entry asynchronously
-        logging.write(Collections.singleton(entry));
+            //Writes the log entry asynchronously
+            logging.write(Collections.singleton(entry));
+        }
+        catch (Exception ex) {
+            throw new InvocationTargetException(ex, "Failed to close Logging instance");
```

Comment:
Should it be "`Failed to write log`" or something alike? I think we relate here to specific case when log cannot be closed, but `try` block might have more cases to fail?